Data/Sys/GameSettings/GW7.ini

```diff
@@ -12,7 +12,12 @@
 [ActionReplay]
```

Comment:
The "(tm)" can be removed on the title of the game, here.

Reply:
That isn't especially important, especially considering it's a pre-existing title.